storage/filesystem/object.go

```diff
@@ -505,3 +506,26 @@ func hashListAsMap(l []plumbing.Hash) map[plumbing.Hash]bool {
 
 	return m
 }
+
+func (s *ObjectStorage) ForEachObjectHash(fun func(plumbing.Hash) error) error {
+	err := s.dir.ForEachObjectHash(fun)
+	if err != nil {
+		if err == storer.ErrStop {
```

Comment:
How can this ever be returned?  I guess you want to support `fun` returning it?  If so, can you document that on `PruneHandler` and the `ForEachObjectHash` interface definition?

Reply:
Because dotgit does import storer. Can handle it there too, but will document this.